apps/challenges/views.py

```diff
@@ -42,6 +42,12 @@ def challenge_list(request, challenge_host_team_pk):
         return paginator.get_paginated_response(response_data)
 
     elif request.method == 'POST':
+
+        if challenge_host_team.created_by != request.user:
```

Comment:
@deshraj i made a check if any object is returned from challengehostteam under the team name and user name . will that be okay ?